.github/workflows/prettybot-automation.yml

```diff
@@ -33,29 +33,32 @@ jobs:
                   git checkout -b ${{ env.BRANCH_NAME }}
 
             - name: Log in to Docker Registry
-              uses: docker/login-action@v2
+              uses: docker/login-action@v3
               with:
                   registry: ${{ secrets.INTERNAL_DOCKER_REGISTRY_URL }}
                   username: ${{ secrets.INTERNAL_DOCKER_READ_ONLY_USER_NAME }}
                   password: ${{ secrets.INTERNAL_DOCKER_READ_ONLY_ACCESS_TOKEN }}
-            
             - name: Run Cleanup Code
               run: |
-                docker run -v ${{ github.workspace }}:/src ${{ secrets.INTERNAL_DOCKER_REGISTRY_URL }}/octopusdeploy/tool-containers/tool-resharper-cli "/scripts/prettybot.sh"
+                chmod -R a+w ${{ github.workspace }}
+                docker run -v ${{ github.workspace }}:/src ${{ secrets.INTERNAL_DOCKER_REGISTRY_URL }}/octopusdeploy/tool-containers/tool-resharper-cli:latest /scripts/prettybot.sh
+            
             - name: Git Check for file changes
               id: git-check
               run: |
-                  if git status --porcelain ; then
-                    echo "changed=true" >> $GITHUB_OUTPUT
+                if git status --porcelain ; then
+                   echo "changed=true" >> $GITHUB_OUTPUT
                   else
                     echo "changed=false" >> $GITHUB_OUTPUT
                   fi
 
             - name: Commit and push changes
               if: steps.git-check.outputs.changed == 'true'
               run: |
-                  git commit -am "Run ReSharper code cleanup"
-                  git push origin ${{ env.BRANCH_NAME }}
+                git config user.email "bob@octopus.com"
```

Comment:
might we use builds@octopus.com here as bob@octopus.com is a fairly generic email

Reply:
I think we'd need to validate if we can merge a commit to this repo (or any other) with another committer identity, given the repo access controls. @edlyn-liew-octo do you mind closing #215 and trying again with a completely dummy git committer?

Reply:
Raised for a seperate conversation in [sc-81385]